### DISCUSSION TITLE
Can't get passed first 10 minutes of Lesson 7. imports reading an error.

### DISCUSSION THREAD

**Discussion by Mrtea228** [Cyfrin/foundry-full-course-cu#395] [General]
I literally followed every step he did in the video starting in Lesson 7 and as soon as I paste the two contracts "FundMe.sol" and "PriceConverter.sol" I get an error on both files with the import. His didn't get an error as i was literally doing everything he did. Not sure why or if i have the wrong set up. 

![error 7202023](https://github.com/Cyfrin/foundry-full-course-f23/assets/138252722/666daefc-33b1-4812-b206-ac24864966bb)

![error 7202023 2](https://github.com/Cyfrin/foundry-full-course-f23/assets/138252722/e3a0a04d-38c8-4ae9-9c43-37c0a730925b)

**Comment by ChefAharoni** [comment]
Your initial import is not correct, you should delete one of the slashes. I'm not sure this will solve your problem if the code, since this contract is probably in the same directory, so the import should start like:
`import {IEcoCoin} from "./IEcoCoin.sol"; `

Also, for the OpenZeppelin error, check [this issue](https://github.com/Cyfrin/foundry-full-course-f23/discussions/271); there's probably a problem in your remappings file (or foundry.toml file).

**Comment by shrutic11** [comment]
I think, you have missed the remappings in the Foundry.toml file.

Please check the following files and if any of these changes make a difference:
1. Foundry.toml

```
[profile.default]
src = "src"
out = "out"
libs = ["lib"]
remappings = ["@chainlink/contracts=lib/chainlink-brownie-contracts/contracts/"]

# See more config options https://github.com/foundry-rs/foundry/tree/master/config
```
2. If you are following the course as is, this is how your import should look like:
PriceConverter.sol
``` solidity
import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```
FundMe.sol
```solidity
import {PriceConverter} from "./PriceConverter.sol";
import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```

**Reply by Mrtea228** [reply]
Thank you. I have the Foundry.toml code right and now copy and pasted the imports here and on his github and both still being marked as error. As soon as i pasted the copy contract from github it showed error but his didn't. Not sure if i don't have a file connected right from previous modules or what. I have no idea why the same code it being marked error. 

The import {PriceConverter} in my FundMe.sol file is working but the import {AggregatorV3Interface} is in error saying file now find. 
The import {AggregatorV3Interface} file is in error also. 
And his video seems to not even explain a possibility.

**Reply by shrutic11** [reply]
Here's what you can try:

One approach would be to download the interface using the following command and make sure you are using the correct syntax and file location in your import statement and your foundry.toml has the correct remappings:
```
forge install smartcontractkit/chainlink-brownie-contract@0.6.1 --no-commit
```
Another approach is to execute the following in the terminal:
```
npm install @chainlink/contracts --save
```
I found similar issue here. The above command seems to have solved the error for them.
https://stackoverflow.com/questions/70683276/how-can-i-import-aggregatorv3interface

**Comment by MansoorAli975** [comment]
Hi
Did you find the solution? I am also facing exactly the same issue

**Reply by EngrPips** [reply]
Please show your error.
